src/main/java/se/sundsvall/oepintegrator/api/WebmessageResource.java

```diff
@@ -79,24 +80,20 @@ ResponseEntity<List<Webmessage>> getWebmessagesByFamilyId(
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
 		@Parameter(name = "instanceType", description = "Which instanceType a message should be sent to", example = "INTERNAL") @PathVariable final InstanceType instanceType,
 		@Parameter(name = "familyId", description = "Family id", example = "123") @PathVariable final String familyId,
-		@Parameter(name = "fromDateTime", description = "The start date and time for filtering web messages (optional)", example = "2024-01-31T12:00:00") @RequestParam(required = false) @DateTimeFormat(
-			iso = DateTimeFormat.ISO.DATE_TIME) final LocalDateTime fromDateTime,
-		@Parameter(name = "toDateTime", description = "The end date and time for filtering web messages (optional).", example = "2024-01-31T12:00:00") @RequestParam(required = false) @DateTimeFormat(
-			iso = DateTimeFormat.ISO.DATE_TIME) final LocalDateTime toDateTime) {
+		@Parameter(name = "fromDateTime", description = "The start date and time for filtering web messages (optional)", example = "2024-01-31T12:00:00") @RequestParam(required = false) @DateTimeFormat(iso = DATE_TIME) final LocalDateTime fromDateTime,
+		@Parameter(name = "toDateTime", description = "The end date and time for filtering web messages (optional).", example = "2024-01-31T12:00:00") @RequestParam(required = false) @DateTimeFormat(iso = DATE_TIME) final LocalDateTime toDateTime) {
 
 		return ok(webmessageService.getWebmessagesByFamilyId(municipalityId, instanceType, familyId, fromDateTime, toDateTime));
 	}
 
-	@GetMapping(path = "/flow-instances/{flowInstanceId}", produces = APPLICATION_JSON_VALUE)
```

Comment:
Is it not more consistent with "/flow-instances/{flowInstanceId}"? We have also "/families/{familyId}" and /attachments/{attachmentId}.

Reply:
It should be "/cases/{flowInstanceId}" in that case, since a flowInstanceId is the case key (see Case resource)

Reply:
I'd say we want to be consistent. I suggest `/cases/{flowInstanceId}`as that matches the Case resource.

Reply:
Added `cases`